exporters/FusionRobotExporter/palette/guide.html

```diff
@@ -1,48 +1,118 @@
 <!DOCTYPE html>
 <html>
-    <style>
-        body {
-            font-family: Arial,sans-serif;
-            margin: 0;
-        }
-        h1 {
-            font-size: 20px;
-            margin: 0;
-            padding: 10px 10px 10px 15px;
-            background-color: #3D3D3D;
-            color: white;
-            font-weight: normal;
-        }
-        h4 {
-            color: GrayText;
-            font-size: 17px;
-            margin-left: 10px;
-            font-weight: normal;
-        }
-        ul {
-            font-size: 18px;
-            padding: 0;
-            margin-left: 40px;
-            margin-right: 30px;
-        }
-        li {
-            margin: 10px 0 10px 0;
-        }
-    </style>
     <head>
-        <link href="css/synthesisExporter.css" rel="stylesheet" type="text/css">
-        <script src="js/fusionInterface.js"></script>
-        <script src="js/utility.js"></script>
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <script src="js/links.js"></script>
+        <style>
+            body {
+                font-family: Arial, sans-serif;
```

Comment:
On fusion, the white background is the same color as the main window, could you make the background slightly darker?